src/common/transformations/tests/common_optimizations/group_normalization_fusion_tests.cpp

```diff
@@ -0,0 +1,570 @@
+class GroupNormalizationFusionTestsFixture_f8e4m3
+    : public GroupNormalizationFusionTestsFixture<element::Type_t::f8e4m3> {};
+class GroupNormalizationFusionTestsFixture_f8e5m2
+    : public GroupNormalizationFusionTestsFixture<element::Type_t::f8e5m2> {};
+class GroupNormalizationFusionTestsFixture_f4e2m1
+    : public GroupNormalizationFusionTestsFixture<element::Type_t::f4e2m1> {};
+class GroupNormalizationFusionTestsFixture_f8e8m0
+    : public GroupNormalizationFusionTestsFixture<element::Type_t::f8e8m0> {};
+
+TEST_P(GroupNormalizationFusionTestsFixture_f16, GroupNormalizationFusionTests_f16) {
+    GroupNormalizationFusionTestsFixture_f16::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_bf16, GroupNormalizationFusionTests_bf16) {
+    GroupNormalizationFusionTestsFixture_bf16::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f32, GroupNormalizationFusionTests_f32) {
+    GroupNormalizationFusionTestsFixture_f32::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_u8, GroupNormalizationFusionTests_u8) {
+    GroupNormalizationFusionTestsFixture_u8::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_u16, GroupNormalizationFusionTests_u16) {
+    GroupNormalizationFusionTestsFixture_u16::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_u32, GroupNormalizationFusionTests_u32) {
+    GroupNormalizationFusionTestsFixture_u32::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_u64, GroupNormalizationFusionTests_u64) {
+    GroupNormalizationFusionTestsFixture_u64::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_i8, GroupNormalizationFusionTests_i8) {
+    GroupNormalizationFusionTestsFixture_i8::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_i16, GroupNormalizationFusionTests_i16) {
+    GroupNormalizationFusionTestsFixture_i16::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_i32, GroupNormalizationFusionTests_i32) {
+    GroupNormalizationFusionTestsFixture_i32::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_i64, GroupNormalizationFusionTests_i64) {
+    GroupNormalizationFusionTestsFixture_i64::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f8e4m3, GroupNormalizationFusionTests_f8e4m3) {
+    GroupNormalizationFusionTestsFixture_f8e4m3::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f8e5m2, GroupNormalizationFusionTests_f8e5m2) {
+    GroupNormalizationFusionTestsFixture_f8e5m2::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f4e2m1, GroupNormalizationFusionTests_f4e2m1) {
+    GroupNormalizationFusionTestsFixture_f4e2m1::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f8e8m0, GroupNormalizationFusionTests_f8e8m0) {
+    GroupNormalizationFusionTestsFixture_f8e8m0::TestBody();
+}
+
+using RawValuesContainer = std::tuple<PartialShape, Shape, Shape, Shape, Shape, unsigned long long, float>;
```

Comment:
```suggestion
using RawValuesContainer = std::tuple<PartialShape,   \\ input0 shape
                                      Shape, \\ output shape
```
Consider add comment about what is purpose of tests parameters.

Reply:
Done